collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/MovingBackwardsTest.kt

```diff
@@ -26,6 +27,15 @@ class MovingBackwardsTest {
             .clickOnString(org.odk.collect.strings.R.string.finalize)
 
             // before disabling moving backward
+            .pressBack(AccessControlPage())
```

Comment:
Could we pull out a helper (maybe just in this class) to handle the common parts of these two tests (before selecting "yes" or "no")?

Reply:
Done.